jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -56,6 +58,16 @@ static GradleProjectProperties getForProject(
     }
   }
 
+  @Nullable
+  static War getWar(Project project) {
```

Comment:
Heads-up: this is going to be called from `GradleLayerConfigurations` (to get the files inside a WAR), so not `private`.